components/actionbutton/themes/express.css

```diff
@@ -21,6 +21,7 @@
 		--spectrum-actionbutton-background-color-hover: var(--spectrum-gray-300);
 		--spectrum-actionbutton-background-color-down: var(--spectrum-gray-400);
 		--spectrum-actionbutton-background-color-focus: var(--spectrum-gray-300);
+		--spectrum-actionbutton-background-color-disabled: var(--spectrum-gray-200);
```

Comment:
Seems odd that disabled and disabled-selected end up being the same color in Express, but I think that's correct 🤷‍♀️ 
![image](https://github.com/user-attachments/assets/89418522-3e7b-4bff-94c8-1cef07b03a0e)

Reply:
I ended up finding some Express Figma files, so I do believe this is correct. https://www.figma.com/design/k5FYSaFhxiLUuPjvWfPSpi/Express-DS--%3E-Atoms?node-id=11-33